src/Model/Notifications.re

```diff
@@ -3,12 +3,16 @@ open Notification;
 
 type t = list(Notification.t);
 
+module ContextMenu =
+  Oni_Components.ContextMenu.Make({});
```

Comment:
The functor implementation is just to generate an identity to connect the anchor with the menu data. It could be generated manually and passed around instead, but this seemed more convenient.